cli/bpmetadata/proto/bpmetadata_ui_ext.proto

```diff
@@ -0,0 +1,180 @@
+syntax = "proto3";
+
+package google.cloud.config.bpmetadata;
+
+option go_package = "github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata";
+
+// ExtensionType specifies the type of extension.
+enum ExtensionType {
+  // UNDEFINED_TYPE indicates that the extension type is undefined.
+  UNDEFINED_TYPE = 0;
```

Comment:
It is usually of form messagetype_undefined
```suggestion
  EXTENSIONTYPE_UNDEFINED = 0;
```

Reply:
done